demisto_sdk/commands/common/logger.py

```diff
@@ -1,142 +1,205 @@
 import logging
-import os
-import sys
+import logging.config
+import os.path
+from logging.handlers import RotatingFileHandler
 from pathlib import Path
-from typing import Optional
 
-from rich.logging import RichHandler
+from demisto_sdk.commands.common.content_constant_paths import CONTENT_PATH
+
+# from rich.logging import RichHandler
+
+LOG_FILE_NAME: str = "demisto_sdk_debug.log"
+
+LOG_FILE_PATH: Path = CONTENT_PATH / LOG_FILE_NAME
+current_log_file_path: str = LOG_FILE_PATH.as_posix()
```

Comment:
why str?

Reply:
Because it failed matching an `str` to a `Path`.